artifacts/definitions/Notebooks/VQLx2.yaml

```diff
@@ -0,0 +1,20 @@
+name: Notebooks.VQLx2
+description: |
+  A notebook initialized with 2 VQL cells
+
+type: NOTEBOOK
+
+sources:
+  - notebook:
+    - type: vql
+      template: |
```

Comment:
You can now specify the output here as well so the message will show up without having to calculate anything

If the output is specified the initial cell will display that and not actually calculate the VQL